tests/cpp-tests/Classes/UITest/CocoStudioGUITest/UIEditBoxTest.cpp

```diff
@@ -340,3 +341,47 @@ bool UIEditBoxTestTextHorizontalAlignment::init() {
   
     return true;
 }
+
+// UIEditBoxTestPressedAndDisabled
+bool UIEditBoxTestPressedAndDisabled::init() {
+    if (!UIScene::init()) {
+        return false;
+    }
+
+    auto glview = Director::getInstance()->getOpenGLView();
+    auto visibleOrigin = glview->getVisibleOrigin();
+    auto visibleSize = glview->getVisibleSize();
+    const auto editBoxSize = Size(visibleSize.width - 100, visibleSize.height * 0.1);
+
+    ui::EditBox* editbox = ui::EditBox::create(editBoxSize + Size(0,40), "extensions/yellow_edit.png", "extensions/orange_edit.png");
+    editbox->setPosition(Vec2(visibleOrigin.x+visibleSize.width/2, visibleOrigin.y+visibleSize.height*3/4));
+    editbox->setFontColor(Color3B::GREEN);
+    editbox->setPlaceHolder("Test Pressed Image");
+    editbox->setPlaceholderFontColor(Color3B::WHITE);
+    editbox->setPlaceholderFontSize(editBoxSize.height/2);
+    editbox->setFontSize(editBoxSize.height/2);
+    editbox->setReturnType(ui::EditBox::KeyboardReturnType::DONE);
+    addChild(editbox);
+
+    editbox = ui::EditBox::create(editBoxSize + Size(0,40), "extensions/yellow_edit.png", "extensions/orange_edit.png", "extensions/green_edit.png");
+    editbox->setPosition(Vec2(visibleOrigin.x+visibleSize.width/2, visibleOrigin.y+visibleSize.height/2));
+    editbox->setFontColor(Color3B::GREEN);
+    editbox->setPlaceHolder("Test Disabled Image - green");
```

Comment:
should added
```
editbox->setEnabled(false);
```
in this test case and next, since you want to test __disable__ situation.